backend/apps/client/src/submission/submission.service.ts

```diff
@@ -238,6 +243,36 @@ export class SubmissionService implements OnModuleInit {
       .padStart(6, '0')
   }
 
+  async checkDelay() {
+    const url =
+      'http://' +
+      this.configService.get('RABBITMQ_HOST') +
+      ':' +
+      this.configService.get('RABBITMQ_API_PORT') +
+      '/api/queues/' +
+      this.configService.get('RABBITMQ_DEFAULT_VHOST') +
+      '/' +
+      this.configService.get('JUDGE_SUBMISSION_QUEUE_NAME')
+
+    const config: AxiosRequestConfig = {
+      method: 'GET',
+      withCredentials: true,
+      auth: {
+        username: this.configService.get('RABBITMQ_DEFAULT_USER') + '',
+        password: this.configService.get('RABBITMQ_DEFAULT_PASS') + ''
+      }
+    }
+    const res = await this.httpService.axiosRef(url, config),
+      threshold = 0.9
```

Comment:
별 거 아니지만 `threshold` 선언 따로하면 좋겠슴다

```suggestion
    const res = await this.httpService.axiosRef(url, config)
    const threshold = 0.9
```